### ISSUE TITLE
styles are broken for NER docs

### ISSUE THREAD

**Issue by kmike** [TeamHG-Memex/eli5#98]
Weight tables don't fit the column at http://eli5.readthedocs.io/en/latest/tutorials/sklearn_crfsuite.html, and there is no scrolling - tables are just truncated.

I was able to fix it locally using custom CSS, but had to revert the fix (https://github.com/TeamHG-Memex/eli5/commit/153ab4aa092746edd54cbe6d802c6bbee833bfb8) because on readthedocs.io original stylesheet was not loaded for some reason.

**Comment by lopuhin** [comment]
hm, RTD applies some hack to make tables look nice (places them in a div with overflow: auto), but it applies it only to the second table. Maaybe if we make all our explanations have one root div (maybe even without styles?), it will work better.

**Comment by lopuhin** [comment]
The same problem in the second table here: http://eli5.readthedocs.io/en/latest/tutorials/sklearn-text.html#debugging-hashingvectorizer

**Comment by kmike** [comment]
For me this [css file](https://github.com/TeamHG-Memex/eli5/blob/master/docs/source/_static/rtd_overrides.css) fixed the issue locally; the problem is that I was not able to load it in readthedocs. Locally it worked fine, but on readthedocs.io it was the only theme css file loaded, so styles were broken.
